ush/ioda/bufr2ioda/marine/b2i/drifter_ioda_variables.py

```diff
@@ -0,0 +1,133 @@
+import numpy as np
+from pyiodaconv import bufr
+from b2iconverter.ioda_variables import IODAVariables
+from b2iconverter.ioda_metadata import IODAMetadata
+from b2iconverter.ioda_addl_vars import IODAAdditionalVariables, compute_seq_num
+from b2iconverter.util import *
+
+
+class DrifterIODAVariables(IODAVariables):
+
+    def __init__(self):
+        self.construct()
+        self.metadata = DrifterMetadata()
+        self.additional_vars = DrifterAdditionalVariables(self)
+
+    def build_query(self):
+        q = bufr.QuerySet()
+        q.add('year', '*/YEAR')
+        q.add('month', '*/MNTH')
+        q.add('day', '*/DAYS')
+        q.add('hour', '*/HOUR')
+        q.add('minute', '*/MINU')
+        q.add('ryear', '*/RCPTIM/RCYR')
+        q.add('rmonth', '*/RCPTIM/RCMO')
+        q.add('rday', '*/RCPTIM/RCDY')
+        q.add('rhour', '*/RCPTIM/RCHR')
+        q.add('rminute', '*/RCPTIM/RCMI')
+        q.add('stationID', '*/RPID')
+        q.add('latitude', '*/CLAT')
+        q.add('longitude', '*/CLON')
+        q.add('depth', '*/DTSCUR/DBSS')
+        q.add('temp', '*/DTSCUR/STMP')
+        q.add('buoy_type', '*/RPSEC4/BUYT')
+        # q.add('ldds', '*/RPSEC4/LDDS')
```

Comment:
any reason for this to remain?

Reply:
Yes, this variable describes the state of the drifters.
It is not clear why we are not using it at this stage, so this is a reminder that it could be used.